.azure-pipelines/mgmt.yml

```diff
@@ -5,31 +5,54 @@ pr:
+      FlpArg: 'Summary;Verbosity=minimal;LogFile=msbuild.sum'
+      FlpArg1: 'warningsonly;logfile=msbuild.wrn'
+      FlpArg2: 'errorsonly;logfile=msbuild.err'
+
+    steps:
+      - task: CmdLine@1
+        inputs:
+          filename: C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.2 Tools\sn.exe
+          arguments: -Vr *,31bf3856ad364e35
+      - task: MSBuild@1
+        inputs:
+          solution: build.proj
+      - task: MSBuild@1
+        inputs:
+          solution: build.proj
+          msbuildArguments: /t:RunTests /clp:ShowTimestamp /flp:$(FlpArg) /flp1:$(FlpArg1) /flp2:$(FlpArg2) /p:IgnorePathTokens="$(IgnoreBuildTokens)"
+      - task: PublishTestResults@2
+        condition: succeededOrFailed()
+        inputs:
+          testResultsFiles: '**/*.trx'
+          testRunTitle: 'Windows Test Results'
+          testResultsFormat: 'VSTest'
+          mergeTestResults: true
 
-steps:
-  - task: CmdLine@1
-    inputs:
-      filename: C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.2 Tools\sn.exe
-      arguments: -Vr *,31bf3856ad364e35
-  - task: MSBuild@1
-    inputs:
-      solution: build.proj
-  - task: MSBuild@1
-    inputs:
-      solution: build.proj
-      msbuildArguments: /t:RunTests /clp:ShowTimestamp /flp:$(FlpArg) /flp1:$(FlpArg1) /flp2:$(FlpArg2) /p:IgnorePathTokens="$(IgnoreBuildTokens)"
-  - task: PublishTestResults@2
-    condition: succeededOrFailed()
-    inputs:
-      testResultsFiles: '**/*.trx'
-      testRunTitle: 'Windows Test Results'
-      testResultsFormat: 'VSTest'
-      mergeTestResults: true
+  - job: Ubuntu_Build
+    pool:
+      vmImage: 'ubuntu-16.04'
+  
+    steps:
+        - task: DotNetCoreInstaller@0
+          displayName: 'Use .NET Core sdk $(DotNetCoreVersion)'
+          inputs:
+            version: '$(DotNetCoreVersion)'
+        - script: './tools/linuxScripts/runNetCoreSdkTests.sh'
+          displayName: 'Build and test on Non-windows machine'
+        - task: PublishTestResults@2
```

Comment:
I don't see any Ubuntu Test Results in the CI job so I must assume there aren't any .trx files produced from the runNetCoreSdkTests.sh script. While this is't a blocker it is something that should be investigated if you guys stay using that script longer term.

Reply:
@weshaggard we have disabled tests on Linux as of today but might enable it at some point, hence the absence of .trx files.